tools/cli/flags.go

```diff
@@ -744,7 +719,7 @@ func getFlagsForQuery() []cli.Flag {
 func getFlagsForStack() []cli.Flag {
 	flags := getFlagsForQuery()
 	for i := 0; i < len(flags); i++ {
-		if flags[i].GetName() == FlagQueryTypeWithAlias {
+		if flags[i].Names()[0] == FlagQueryType {
```

Comment:
it's kinda awkward looking, but 👍 yeah all the built-in ones seem to return 1 or more in all cases.  and I'm not seeing any better option (without making things much more complex, which is probably worse).